core/classifiers/classifier_utils.py

```diff
@@ -105,18 +105,19 @@ def convert_float_numbers_to_string_in_classifier_data(classifier_data):
             an exception is raised to report the error. The classifier data
             must not include any string values which can be casted to float.
         Exception. If classifier data contains an object whose type is other
-            than integer, string, dict, floats or list.
+            than integer, string, dict, float or list.
 
     Returns:
         dict|list|string|int|float. Modified classifier data in which float
             values are converted into strings.
     """
     if isinstance(classifier_data, dict):
+        classifier_data_with_stringified_floats = {}
         for k in classifier_data:
-            classifier_data[k] = (
+            classifier_data_with_stringified_floats[k] = (
                 convert_float_numbers_to_string_in_classifier_data(
                     classifier_data[k]))
-        return classifier_data
+        return classifier_data_with_stringified_floats
     elif isinstance(classifier_data, list):
         new_list = []
```

Comment:
This should also be classifier_data_with_stringified_floats

Reply:
Done